lib/fs.js

```diff
@@ -1940,8 +1940,7 @@ WriteStream.prototype.open = function() {
 
 
 WriteStream.prototype._write = function(data, encoding, cb) {
-  if (!(data instanceof Buffer))
-    return this.emit('error', new Error('Invalid data'));
+  // `data` is always a Buffer here.
```

Comment:
Is this comment going to go stale when we become more liberal in accepting Uint8Arrays?